test/windows/experimental.spec.ts

```diff
@@ -0,0 +1,127 @@
+      path: imageNameAndTag,
+    });
+
+    const depGraph: DepGraph = pluginResult.scanResults[0].facts.find(
+      (fact) => fact.type === "depGraph",
+    )!.data;
+    expect(depGraph.rootPkg.name).toEqual("docker-image|nginx.tar");
+    expect(depGraph.rootPkg.version).toBeUndefined();
+
+    const imageId: string = pluginResult.scanResults[0].facts.find(
+      (fact) => fact.type === "imageId",
+    )!.data;
+    expect(imageId).toEqual(
+      "5a3221f0137beb960c34b9cf4455424b6210160fd618c5e79401a07d6e5a2ced",
+    );
+    expect(pluginResult.scanResults[0].identity.type).toEqual("deb");
+    expect(
+      depGraph.getDepPkgs().find((dep) => dep.name === "adduser"),
+    ).toBeDefined();
+
+    const imageLayers: string[] = pluginResult.scanResults[0].facts.find(
+      (fact) => fact.type === "imageLayers",
+    )!.data;
+    expect(imageLayers).toEqual([
+      path.normalize(
+        "ac415f8e415b242117277e7ee5224b30389698b46101e0f28224490af3b90a9d/layer.tar",
+      ),
+    ]);
+    expect(pluginResult.scanResults[0].identity.args?.platform).toEqual(
+      "linux/amd64",
+    );
+  });
+
+  it("can scan oci-archive image type", async () => {
+    const fixturePath = getFixture("oci-archives/alpine-3.12.0.tar");
+    const imageNameAndTag = `oci-archive:${fixturePath}`;
+
+    const pluginResult = await plugin.scan({
+      path: imageNameAndTag,
+    });
+
+    const depGraph: DepGraph = pluginResult.scanResults[0].facts.find(
+      (fact) => fact.type === "depGraph",
+    )!.data;
+    expect(depGraph.rootPkg.name).toEqual("docker-image|alpine-3.12.0.tar");
+    expect(depGraph.rootPkg.version).toBeUndefined();
+    const imageId: string = pluginResult.scanResults[0].facts.find(
+      (fact) => fact.type === "imageId",
+    )!.data;
+    expect(imageId).toEqual(
+      "sha256:0f5f445df8ccbd8a062ad3d02d459e8549d9998c62a5b7cbf77baf68aa73bf5b",
+    );
+    expect(pluginResult.scanResults[0].identity.type).toEqual("apk");
+    expect(
+      depGraph
+        .getDepPkgs()
+        .find((dep) => dep.name === "alpine-keys/alpine-keys"),
+    ).toBeDefined();
+    const imageLayers: string[] = pluginResult.scanResults[0].facts.find(
+      (fact) => fact.type === "imageLayers",
+    )!.data;
+    expect(imageLayers).toEqual([
+      path.normalize(
+        "sha256:df20fa9351a15782c64e6dddb2d4a6f50bf6d3688060a34c4014b0d9a752eb4c",
+      ),
+    ]);
+  });
+
+  it("can static scan for Identifier type image (nginx:1.19.0)", async () => {
+    const imageNameAndTag = "nginx:1.19.0";
+
+    await expect(() =>
+      plugin.scan({
+        path: imageNameAndTag,
+      }),
+    ).rejects.toEqual(
+      new Error("The image does not exist for the current platform"),
+    );
+  });
+
+  it("can static scan for Identifier type image (python:3.9.0rc2-slim by SHA256)", async () => {
+    const imageNameAndTag =
+      "python@sha256:8f9c0e68b4c7ddcd1dd5ac222572c0fec9471212a0ab2a32e1a340e436fdc46f";
```

Comment:
This is new! Seems like the previous 3.10 tag got updated every once in a while and even the image SHAs disappeared from Docker Hub